FizzBuzz.py

```diff
@@ -1,27 +1,42 @@
-from types import ListType, IntType
+"""Print all integers from 1 to 100. However, for multiples of three,
+print 'Fizz' instead of the number, for multiples of five, print
+'Buzz', and for numbers which are multiples of both three and five,
+print 'FizzBuzz'.
+"""
+def is_appropriate(number):
```

Comment:
i'll test tomorrow at work, for now, change is_appropriate to a pure predicate (returns true/false, does not assert) and switch if statement uses of it to assertions